proto/plan.proto

```diff
@@ -268,6 +268,18 @@ message ViewDef {
 	string view  = 1;
 }
 
+message ComputeIndexDef {
+	repeated string names = 1;
+	repeated string table_names = 2;
+	repeated bool uniques = 3;
+}
+
+message ComputeIndexInfo {
+	string table_name = 1;
+	repeated string attrs = 2;
+	repeated ColDef cols = 3;
+}
+
```

Comment:
Why do we need these redundant and separate structures? I don't think it's necessary.

Reply:
It is ineffective to scan all the defs, so we need to keep index infos